src/mame/misc/marywu.cpp

```diff
@@ -134,7 +134,7 @@ void marywu_state::ay2_port_b_w(uint8_t data)
 
 void marywu_state::p3_port_w(uint8_t data) // 1 led are used.
 {
-	for (uint8_t i = 0; i < 8; i++)
+	for (uint8_t i = 0; i < 0; i++)
```

Comment:
This causes the loop to be executed zero times, defeating the whole purpose of adding this as a callback.

Since external memory is hooked up to the MOVX space, it's unlikely P36 and P37 are used to drive LEDs, but some of the lower 6 bits might be.